libretro/libretro.c

```diff
@@ -856,6 +851,13 @@ void update_variables()
             u_cbutton = RETRO_DEVICE_ID_JOYPAD_X;
     }
 
+    var.key = CORE_NAME "-alt-map";
+    var.value = NULL;
+    if (environ_cb(RETRO_ENVIRONMENT_GET_VARIABLE, &var) && var.value)
+    {
+        alternate_mapping = strcmp(var.value, "False");
```

Comment:
alternate_mapping = !strcmp(var.value, "False") ? 0 : 1;

Reply:
Sure. What I had works, but I can also appreciate wanting consistency with the rest of the file.